exercises/ruby-1-a/ruby_1_a_example.rb

```diff
@@ -1,11 +1,17 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
+class ShoppingCart
+  def initialize(items = {})
+    @items = items
   end
-  word_vowels.sort_by {|w,c| -c}.first.first
+
+  def total_amount
+    items.sum(0) { |key, value| value }
```

Comment:
item, cost`
```suggestion
    items.sum(0) { |_item, cost| cost }
```

It is idiomatic to prefix unused variables with an underscore, or as an underscore completely.